package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "framework-arduinoespressif8266",
+    "description": "Arduino Wiring-based Framework (ESP8266 Core)",
+    "url": "https://github.com/esp8266/Arduino",
+    "version": "2.4.0-rc.1"
```

Comment:
`2.4.0-dev` is good, but need to update it to the final release if you make release tag.

Currently, we have own forked and patched ESP826 Core for Arduino. Our community does not like it and asks us to use your repository. That is why I added `package.json` manifest. It's required for PIO Package Manager. After merge/release, I'll switch our development platform to this repo [here](https://github.com/platformio/platform-espressif8266/blob/develop/platform.json#L37:L40).